src/main/java/com/epam/upskill/service/file/FileWorker.java

```diff
@@ -1,13 +1,20 @@
-package com.epam.upskill.service;
+package com.epam.upskill.service.file;
 
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.StandardCopyOption;
 
 public class FileWorker {
+
+    private static final Logger logger = LogManager.getLogger(FileWorker.class);
```

Comment:
logger is constant so 
```suggestion
    private static final Logger LOGGER = LogManager.getLogger(FileWorker.class);
```